influent-java/src/main/java/influent/forward/ForwardClientNode.java

```diff
@@ -0,0 +1,46 @@
+package influent.forward;
+
+import influent.internal.util.InetNetwork;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.List;
+
+public class ForwardClientNode {
```

Comment:
I'm understanding that `node` comes from `in_forward.rb`.
But I feel that meanings of `ForwardClient` and `ForwardClientNode` are not clear……
I'm considering good names.